internal/app/go-filecoin/node/node.go

```diff
@@ -465,12 +470,28 @@ func (node *Node) setupStorageMining(ctx context.Context) error {
 		return errors.Wrap(err, "error initializing storage protocol")
 	}
 
-	// TODO: Retrieval Market Integration
```

Comment:
this was left in the func to set up storage mining.  instead put the initialization in its own func, which will be called in a later PR.